sunpy/net/dataretriever/tests/test_norh.py

```diff
@@ -1,41 +1,135 @@
+from unittest import mock
+from datetime import datetime, timedelta
+
 import pytest
+from hypothesis import given, settings, HealthCheck
 
 import astropy.units as u
 from astropy.time import TimeDelta
 from astropy.time import Time
-
-from sunpy.time.timerange import TimeRange
-from sunpy.net.dataretriever.client import QueryResponse
 import sunpy.net.dataretriever.sources.norh as norh
+from sunpy.net.download import Results
+from sunpy.net.tests.strategies import time_attr, range_time
+from sunpy.net.dataretriever.client import QueryResponse
 from sunpy.net.fido_factory import UnifiedResponse
 from sunpy.net import Fido
 from sunpy.net import attrs as a
+from sunpy.time import parse_time
+from sunpy.time.timerange import TimeRange
 
-from hypothesis import given, settings, HealthCheck
-
-from sunpy.net.tests.strategies import time_attr, range_time
+NORHClient = norh.NoRHClient()
+BASEURL = 'ftp://anonymous:data@sunpy.org@solar-pub.nao.ac.jp/pub/nsro/norh/data/tcx/%Y/%m/{freq}%y%m%d'
+
+def create_url(start, end, wavelength):
```

Comment:
Hey,
I wanted to generate a list of URLs instead of making an online request. I intended to use the scraper for the above purpose but then I wanted to get a list of URLs that would be scraped, and not the scraped response.

Reply:
If I remember right you can create them with a particular time...

Reply:
Yeah, that is true. 
However, using the `util/scraper.py` returns a list of Response, and not a list of URLs. Since I am using the helper function in creating the list of urls for offline tests, it is not possible to use `util/scraper`